### PR TITLE
🌐 Add Korean Translation for `docs/ko/docs/advanced/response-cookies.md`

### PR DESCRIPTION
**Pull request by kim-sangah**
Add translation to docs/ko/docs/advanced/response-cookies.md

### CODE REVIEW

File: docs/ko/docs/advanced/response-cookies.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# 응답 쿠키
+
+## `Response` 매개변수 사용하기
+
+*경로 작업 함수*에서 `Response` 타입의 매개변수를 선언할 수 있습니다.
```
Comment (sodaMelon) [inline docs/ko/docs/advanced/response-cookies.md:5]:
```suggestion
*경로 작동 함수*에서 `Response` 타입의 매개변수를 선언할 수 있습니다.
```
here is the reference for Path operation in korean terms
https://github.com/fastapi/fastapi/discussions/3167

File: docs/ko/docs/advanced/response-cookies.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+그런 다음 해당 *임시* 응답 객체에서 쿠키를 설정할 수 있습니다.
+
+```Python hl_lines="1  8-9"
+{!../../docs_src/response_cookies/tutorial002.py!}
+```
+
+그런 다음 필요한 객체(예: `dict`, 데이터베이스 모델 등)를 반환할 수 있습니다.
```
Comment (sodaMelon) [inline docs/ko/docs/advanced/response-cookies.md:13]:
```suggestion
그런 다음 필요한 객체(`dict`, 데이터베이스 모델 등)를 반환할 수 있습니다.
```
It seems unnecessary.

File: docs/ko/docs/advanced/response-cookies.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+그리고 `response_model`을 선언했다면 반환한 객체를 필터링하고 변환하는 데 여전히 사용됩니다.
+
+**FastAPI**는 그 *임시* 응답에서 쿠키(또한 헤더 및 상태 코드)를 추출하고, 반환된 값이 포함된 최종 응답에 이를 넣습니다. 이 값은 `response_model`로 필터링됩니다.
+
+또한 종속성에서 `Response` 매개변수를 선언하고, 해당 종속성에서 쿠키(및 헤더)를 설정할 수도 있습니다.
```
Comment (sodaMelon) [inline docs/ko/docs/advanced/response-cookies.md:19]:
```suggestion
또한 의존관계에서 `Response` 매개변수를 선언하고, 해당 의존성에서 쿠키(및 헤더)를 설정할 수도 있습니다.
```
의존성 또는 의존관계
중에 어울리는 말을 써주세요
그냥  dependencies라는 단어를 그대로 써도 될 것 같기도하고, 🤔🤔🤔🤔 어렵네요.

File: docs/ko/docs/advanced/response-cookies.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## `Response`를 직접 반환하기
+
+코드에서 `Response`를 직접 반환할 때도 쿠키를 생성할 수 있습니다.
+
+이를 위해 [Response를 직접 반환하기](response-directly.md){.internal-link target=_blank}에서 설명한 대로 응답을 생성할 수 있습니다.
+
+그런 다음 쿠키를 설정하고 반환하면 됩니다:
+Python hl_lines="10-12"
```
Comment (sodaMelon) [inline docs/ko/docs/advanced/response-cookies.md:28]: could you check the new created documentation? 
it is broken from here.

**Comment by sodaMelon** [comment]
hi i left some comment for PR.
please check it when you are not busy

File: docs/ko/docs/advanced/response-cookies.md (hunk 4 above)
Comment (kim-sangah) [reply docs/ko/docs/advanced/response-cookies.md]:
``` 표시 추가해서 변경했습니다!

**Review by sodaMelon — APPROVED** [review]
LGTM 👍

File: docs/ko/docs/advanced/response-cookies.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+그런 다음 해당 *임시* 응답 객체에서 쿠키를 설정할 수 있습니다.
+
+```Python hl_lines="1  8-9"
+{!../../docs_src/response_cookies/tutorial002.py!}
+```
+
+그런 다음 필요한 객체(`dict`, 데이터베이스 모델 등)를 반환할 수 있습니다.
+
+그리고 `response_model`을 선언했다면 반환한 객체를 필터링하고 변환하는 데 여전히 사용됩니다.
```
Comment (JaeHyuckSa) [inline docs/ko/docs/advanced/response-cookies.md:15]:
```suggestion
그리고 `response_model`을 선언했다면 반환한 객체를 거르고 변환하는 데 여전히 사용됩니다.
```

File: docs/ko/docs/advanced/response-cookies.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+**FastAPI**는 그 *임시* 응답에서 쿠키(또한 헤더 및 상태 코드)를 추출하고, 반환된 값이 포함된 최종 응답에 이를 넣습니다. 이 값은 `response_model`로 필터링됩니다.
```
Comment (JaeHyuckSa) [inline docs/ko/docs/advanced/response-cookies.md:17]:
```suggestion
**FastAPI**는 그 *임시* 응답에서 쿠키(또한 헤더 및 상태 코드)를 추출하고, 반환된 값이 포함된 최종 응답에 이를 넣습니다. 이 값은 `response_model`로 걸러지게 됩니다.
```

File: docs/ko/docs/advanced/response-cookies.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+```Python hl_lines="1  18"
+{!../../docs_src/response_directly/tutorial002.py!}
+```
+/// tip
+
+`Response` 매개변수를 사용하지 않고 응답을 직접 반환하는 경우, FastAPI는 이를 직접 반환한다는 점에 유의하세요.
+
+따라서 데이터가 올바른 유형인지 확인해야 합니다. 예: `JSONResponse`를 반환하는 경우, JSON과 호환되는지 확인하세요.
+
+또한 `response_model`로 필터링되어야 할 데이터를 보내지 않는지 확인하세요.
```
Comment (JaeHyuckSa) [inline docs/ko/docs/advanced/response-cookies.md:37]:
```suggestion
또한 `response_model`로 걸러져야 할 데이터가 전달되지 않도록 확인하세요.
```

**Review by JaeHyuckSa — APPROVED** [review]

**Comment by alejsdev** [comment]
Thanks! @kim-sangah :tada: 
And thanks for the reviews @sodaMelon @JaeHyuckSa :nerd_face:

**Pull request merged**